src/main/java/com/amazonaws/kinesisvideo/producer/KvsRetryStrategyType.java

```diff
@@ -0,0 +1,25 @@
+package com.amazonaws.kinesisvideo.producer;
+
+public enum KvsRetryStrategyType {
+    
+    DISABLED(0), EXPONENTIAL_BACKOFF_WAIT(1), UNKNOWN(-1);
+
+    private final int mValue;
+
+    public static final int getRetryStrategyType(String type) {
+        for (KvsRetryStrategyType strategyType : KvsRetryStrategyType.values()) {
+            if (strategyType.name().equals(type)) {
```

Comment:
Can we use the `intValue()` instead of comparing strings with `name()`?

Reply:
The input to this function is a string so we have to search by string equivalent